part-4/model-building-components.md

```diff
@@ -677,15 +677,15 @@ First, an active transporter molecule needs to be defined:
 6. Click on the transporter molecule at the top level of the molecules tree to modify this molecule's parameters, as described above in, [Molecule Parameters](model-building-components.md#molecule-parameters). This may be the initial amount of transporter or a concentration parameter.
 7. Right-click on the transporter attached to the molecule to be transported, and select **Create Transport** from the context menu. A window named "New Active Transport" opens.
-8. Enter a name into the Name input box, like "PGP Transport". Then, follow the steps described in the previous section “Passive Transports” for selecting source and target, define a transport rate parameter and entering a transport kinetics formula.
+8. Enter a name into the Name input box, like "TRANS Transport". Then, follow the steps described in the previous section “Passive Transports” for selecting source and target, define a transport rate parameter and entering a transport kinetics formula.
 9. The kinetics formula of an active transport process is entered into the formula input box within the Tab **Kinetic** so that the red error symbol ![](../assets/icons/ErrorProvider.svg) will disappear. A typical active transport formula will be dependent on the transporter concentration, substrate concentration in source and target container, and on molecule specific parameters, like a KM value for substrate and transporter. You will need to add all the required concentrations and parameters as references, or you may enter them in numeric form into the equation.
 
-Continuing with our **example project**, let us enter a transport called "PGP" for molecule "A" and a transport process called "PGP Transport A" which runs in the opposite direction of the above passive transport, i.e., from "Vial2" as source to "Vial1" as target. As references for the transport equation, you need the concentration parameters of "PGP" and of "A" from the references tree. The alias of the PGP concentration is renamed to "C\_PGP", and that of molecule "A" to "C\_A" by just overriding the default names. The equation to be entered is "0.001\*C\_PGP\*C\_A". The figure below shows what the screen should look like after everything is properly set up.
+Continuing with our **example project**, let us enter a transport called "TRANS" for molecule "A" and a transport process called "TRANS Transport A" which runs in the opposite direction of the above passive transport, i.e., from "Vial2" as source to "Vial1" as target. As references for the transport equation, you need the concentration parameters of "TRANS" and of "A" from the references tree. The alias of the TRANS concentration is renamed to "C\_TRANS", and that of molecule "A" to "C\_A" by just overriding the default names. The equation to be entered is "0.001\*C\_TRANS\*C\_A". The figure below shows what the screen should look like after everything is properly set up.
 
 ![Active Transport has been entered](../assets/images/part-4/active-transport-entered.jpg)
```

Comment:
the screenshot must be updated then as well...
![grafik](https://user-images.githubusercontent.com/25061876/162149952-2b87fc72-9870-46ae-9038-b9e6d9f93565.png)